sample-components-prefab/src/global-scope.js

```diff
@@ -0,0 +1,2 @@
+globalThis.process = { env: { NODE_ENV:"development" } };
```

Comment:
Add a comment here explaining that we're not actually as node.js runtime?  I assume this was needed to trick some node detection somewhere inside react?

Reply:
added comment